includes/api/ApiQueryUserContributions.php

```diff
@@ -90,23 +75,14 @@ public function execute() {
 		// Fetch each row
 		foreach ( $res as $row ) {
 			if ( ++ $count > $limit ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this increment or decrement operator into a dedicated statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS881)

[project-id]: wikia:app